### PR TITLE
VAR-23 | Show warning message for IE11 users

### PR DESCRIPTION
**Pull request by Rasbin**
This PR is for the issue #**VAR-23**
 
* Shows warning message to IE users about the page not displaying in their browser.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -22,10 +23,47 @@ const finalState = Immutable(initialStoreState).merge([initialServerState, initi
...
-  <Provider store={store}>
-    <Router>{getRoutes()}</Router>
-  </Provider>,
-  document.getElementById('root')
-);
+if (browserName === 'IE') {
+  render(
+    <div>
+      <p style={warningStyle}>
+        Currently, Varaamo does not support IE11.
```
Comment (Nizar-Rahme) [inline app/index.js]: That is no longer true after adding the polyfill.

File: app/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
+import 'react-app-polyfill/ie11';
```
Comment (Nizar-Rahme) [inline app/index.js]: I'm quite sure that there will be some broken styles in IE, have checked it out?

Comment (Rasbin) [reply app/index.js]: There are some broken stuffs in IE, but in this PR the task is to only display the warning message to IE users. There is another task for fixing stuffs in IE.

File: app/index.js [hunk 3]
```diff
@@ -22,10 +23,47 @@ const finalState = Immutable(initialStoreState).merge([initialServerState, initi
   deep: true,
 });
 const store = configureStore(finalState);
+const warningStyle = {
+  margin: '20px',
```
Comment (midori0507) [inline app/index.js]: no inline style please, add a className in sass and put styling in it

File: app/index.js [hunk 4]
```diff
@@ -22,10 +23,47 @@ const finalState = Immutable(initialStoreState).merge([initialServerState, initi
...
+  border: '2px solid #faebcc',
+};
 
-render(
-  <Provider store={store}>
-    <Router>{getRoutes()}</Router>
-  </Provider>,
-  document.getElementById('root')
-);
+if (browserName === 'IE') {
```
Comment (midori0507) [inline app/index.js]: Write unit test

File: app/index.js (hunk 1 above)
Comment (Rasbin) [reply app/index.js]: The error message `div` is getting rendered in IE browser and normally as before in other browsers only after adding the polyfill. Without polyfill, I am just getting the blank screen in IE.

**Pull request merged**